recipe/meta.yaml

```diff
@@ -11,10 +11,13 @@ source:
   patches:
     # http://lists.gnu.org/archive/html/bug-ncurses/2011-04/txtkWQqiQvcZe.txt
     - fix.patch
+    - ncurses-5.9-gcc-5.patch
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 5 # [linux64]
+  number: 6 # [linux32]
+  number: 1 # [osx]
```

Comment:
No problem. (It looked like a copy-n-paste thing :wink:)

Our builds for this package are in the `0`th, you are changing and bumping to `1`. We don't use individual platform build numbers to avoid the extra burden on the maintainers. (That means even if your change only affects OS X we bump all the build numbers.)